config/manager/manager.yaml

```diff
@@ -33,6 +33,12 @@ spec:
         name: manager
         securityContext:
           allowPrivilegeEscalation: false
+          readOnlyRootFilesystem: true
+          runAsNonRoot: true
+          capabilities:
+            drop: [ "ALL" ]
```

Comment:
Slight side note, do you know what they mean w/ the `| index("all")` ? Looks like sort of an assertion 

https://kubesec.io/basics/containers-securitycontext-capabilities-drop-index-all/

Reply:
It looks like a selector language, I'm not sure precisely which one it is tho'